api_v2.json

```diff
@@ -1732,11 +1733,38 @@
 						"description": "**POST /v2/payments/{id}/delete ** \n   Delete a prevoiusly created payment and returns a hash containing the details of the deleted payment.\n\n\n**Required Fields**\n\nid:Unique ID of payment\n\n**Optional Fields**\n\non\\_behalf\\_of: contact ID\n\n**Response**\n\nId:Payment unique ID\n\nPayer\\_id:Unique payer ID\n\nAmount:Amount of Payment to 2dp\n\nBeneficiary\\_id:ID of the beneficiary\n\nCurrency:3 character ISO 4217 currency code\n\nConversion\\_id:Conversion unique ID\n\nReference:Reference\n\nPayment\\_type:Can be standard or local\n\nPayment\\_date:ISO 8601 Date when the payment should be paid\n\nreason\t:Reason for payment\n\nstatus:Status of the payment\n\ntransferred\\_at:ISO 8601 Date when the Payment was sent\n\nauthorisation\\_steps\\_required:Number of authorisation steps required\n\ncreator\\_contact\\_id:ID of the contact\n\nlast\\_updater\\_contact\\_id:ID of the contact\n\nshort\\_reference:transfer ID of the payment\n\nfailure\\_reason:Reason for the failure\n\ncreated\\_at:Datetime of when the Payment was created\n\nupdated\\_at:Datetime of when the Payment was last updated\n\n\n"
 					},
 					"response": []
+				},
+				{
+					"name": "Payment Confirmation",
+					"request": {
+						"method": "GET",
+						"header": [],
+						"body": {},
+						"url": {
+							"raw": "192.168.30.159:36016/v2/payment/test/confirmation",
+							"host": [
+								"192",
+								"168",
+								"30",
+								"159"
+							],
+							"port": "36016",
+							"path": [
+								"v2",
+								"payment",
+								"test",
+								"confirmation"
+							]
+						},
+						"description": "**GET /v2/payment/{id}/confirmation**\nPayment Confirmation API. It provides ability to get information about specific payment confiramiton entity, based on id.\n\n**Required Fields**\n\nid:Unique ID of payment confirmation\n\n**Response**\n\n  Id:Payment confirmation confirmation unique ID\n  \n  Short reference:the Payment confirmation short reference\n  \n  Status:Payment confirmation status\n  \n  Expires_at:Payment confirmation expiration date and time\n  \n  Confirmation_url: The url to the payment confirmation file - this should be available only when the status is 'completed'\n  \n  Account_id:The account id where the payment confirmation and the payment belong to\n  \n  Created_at:The date time when the confirmation was created\n  \n  Updated_at:The date time of the latest status update of the confirmation)"
```

Comment:
typo

Reply:
also status can be `failed` or `completed`

Reply:
also we fetching payment_confirmation by payment id, not by payment_confirmation id